src/main/webapp/js/userMap.js

```diff
@@ -14,20 +14,22 @@ function handleData(err, countryCoordinates, userData) {
         handleError();
         return;
     }
-    var userCountries = Object.keys(userData);
     var countriesArr = [];
     var total = 0;
+    var date = userData.lastUpdated;
+    var userCountries = Object.keys(userData.institutes);
```

Comment:
@wkurniawan07 i din't get what you are saying? I have removed the `var userCountries = Object.keys(userData)` this.SO what exactly should i need to reuse ?
`